### PR TITLE
OLS-878: add stub topic that points to standalone content

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): OCP 4.15+

Issue: https://issues.redhat.com/browse/OLS-878

Link to docs preview:
https://79501--ocpdocs-pr.netlify.app/openshift-enterprise/latest/lightspeed/about/ols-openshift-lightspeed-overview

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: lightspeed/about/ols-about-openshift-lightspeed.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-lightspeed"]
+include::_attributes/common-attributes.adoc[]
+= Lightspeed overview
+:context: about-lightspeed
+
+toc::[]
+
+{ols-official} applies generative AI to deploying and scaling both traditional and cloud-native applications on {product-title} clusters, helping novice users to more quickly build skills to run the application platform and helping experts to be more efficient.
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply lightspeed/about/ols-about-openshift-lightspeed.adoc]: updated.

File: lightspeed/about/ols-openshift-lightspeed-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="openshift-lightspeed-overview"]
+include::_attributes/common-attributes.adoc[]
+= OpenShift Lightspeed overview
+:context: openshift-lightspeed-overview
+
+toc::[]
+
+{ols-official} provides intelligent, natural language processing capabilities designed to make Red Hat cloud-native application platforms easier to use for novices users and more efficient for experienced professionals.
```
Comment (gabriel-rh) [inline lightspeed/about/ols-openshift-lightspeed-overview.adoc:9]: for novices users   -> for novice users

**Review by gabriel-rh — COMMENTED** [review]
lgtm

**Review by gabriel-rh — APPROVED** [review]
LGTM

File: lightspeed/about/ols-openshift-lightspeed-overview.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply lightspeed/about/ols-openshift-lightspeed-overview.adoc]: done

**Pull request merged**